docs/en/setup/service-agent/java-agent/containerization.md

```diff
@@ -54,3 +54,5 @@ spec:
         - name: JAVA_TOOL_OPTIONS
           value: "-javaagent:/skywalking/agent/skywalking-agent.jar"
 ```
+
+To use the java agent more natively, you can try the [java agent injector](https://github.com/apache/skywalking-swck/blob/master/docs/java-agent-injector.md) to inject the agent image as a sidecar.
```

Comment:
I think this would be our recommended method whenever applicable, how about moving this to the top of this section (`## Kubernetes`)?

Reply:
If you are going to move on the top, please add statements like, 

Currently, SkyWalking provides two ways to install agents on your services on the Kubernetes,
1. xxxx
2. xxxx

Reply:
Good idea.